components/Footer.jsx

```diff
@@ -5,12 +5,15 @@ import onsLogo from "../public/images/ons-logo-white-background.png";
 export default function Footer() {
   return (
     <>
-      <footer className="text-xs pt-6 pb-8 px-11 flex justify-around items-center flex-wrap border-solid border-ecnmy-charcoal border-t-2 border-b-2 mt-10">
-        <div className="flex flex-col items-start gap-4">
-          <p className="text-[16px] py-1 font-semibold text-center w-full">
-            Data Sources: This dashboard represents local data available from a
-            range of sources, current sources are
+      <footer className="text-xs pt-6 pb-8 px-11 flex justify-around items-center flex-wrap border-solid border-ecnmy-charcoal border-t-2 border-b-2">
+        <div className="flex flex-row flex-wrap items-start gap-4">
+          <p className="text-[16px] py-1 font-medium max-w-xs mb-2 flex flex-wrap">
+            <span className="font-bold">Data Sources</span>
```

Comment:
Maybe change the p to a div then have Data Sources as a title (<h2></h2>) and the rest of the text within a nested p tag. So

<div>
<h2>Data Sources</h2>
<p>This dashboard represents local data available from a range of sources. Current sources include the following:</p>
</div>